packages/bpk-component-chip/src/BpkSelectableChip.tsx

```diff
@@ -41,7 +42,10 @@ const BpkSelectableChip = ({
   className,
   disabled = false,
   dismissible = false,
+  id = null,
   leadingAccessoryView = null,
+  name = '',
+  onClick = ()=>{},
```

Comment:
Maintain existing onClick to not break any changes - hook it into the onChange event. Which toggles on Click.